pyproject.toml

```diff
@@ -17,7 +17,9 @@ classifiers = [
 panimg = "panimg.cli:cli"
 
 [tool.poetry.dependencies]
-python = "^3.8"
+# Change back to "^3.8" after https://github.com/imi-bigpicture/wsidicom/pull/96
+# is merged and released
+python = ">=3.8,<3.12"
```

Comment:
We cannot release with such a restriction, we would create the same issue for all users of this library.

Reply:
Then we need to either wait for a new release of wsidicom or fork it and release a fix ourselves.

Reply:
So long as they don't have a complex build system you should be able to install directly for now using a git source and tag, then make a comment to replace it when they put a new version on pypi.